internal/handlers/common/aggregations/operators/sum.go

```diff
@@ -0,0 +1,172 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package operators provides aggregation operators.
```

Comment:
I see that we already have a package-level comment in `operators.go`.
Since there is the same comment in the `type.go`, perhaps it's OK to leave it as that for now and then remove both of them and keep the extended package comment from `operators.go`?